en/docs/install-and-setup/setup/kubernetes-deployment/kubernetes/am-pattern-4-acp-tm-gw-km.md

```
@@ -1,752 +1,981 @@
 
-  ```bash
-  helm install envoy-gateway oci://docker.io/envoyproxy/gateway-helm \
-    --version v1.7.0 -n envoy-gateway-system \
-    --set config.envoyGateway.extensionApis.enableBackend=true \
-    --set envoyGateway.gateway.experimentalFeatures.enabled=true \
-    --create-namespace
-  ```
+1. Extract the default keystores from your ACP image and create the secret:
 
-- Create and apply Gateway and GatewayClass resources.
+    ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)